test/routes/player_route_test.dart

```diff
@@ -39,15 +46,46 @@ void main() {
     });
   });
 
-  testWidgets('can open queue screen by swiping to left', (WidgetTester tester) async {
-    await tester.runAppTest(() async {
-      // Expand the route
-      await expandPlayerRoute(tester);
+  group('queue screen', () {
+    testWidgets('can open by swiping to left', (WidgetTester tester) async {
+      await tester.runAppTest(() async {
+        QueueControl.instance.resetQueue();
```

Comment:
Because I'm changing the type of the queue in the new test, and this test assumes that the queue is a default queue. The test should make sure the start state is as expected.

Reply:
> The test should make sure the start state is as expected.

This is guaranteed by using `runAppTest`

Reply:
(and also by using `setUpAppTest`)